ci/packer/vmbase.json

```diff
@@ -0,0 +1,41 @@
+{
+  "builders": [
+    {
+      "type": "qemu",
+      "iso_urls": [
+        "https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-10.6.0-amd64-netinst.iso"
+      ],
+      "boot_command": [
+        "<esc><wait>",
+        "install <wait>",
+        "auto=true <wait>",
+        "netcfg/get_hostname={{ .Name }} <wait>",
+        "netcfg/get_domain=foo <wait>",
+        "debconf/frontend=noninteractive <wait>",
+        "url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed.cfg <wait>",
+        "<enter><wait>"
+      ],
+      "iso_checksum": "sha256:2af8f43d4a7ab852151a7f630ba596572213e17d3579400b5648eba4cc974ed0",
+      "output_directory": "debian_base",
+      "http_directory": ".",
+      "shutdown_command": "shutdown -P now",
+      "disk_size": "5000M",
```

Comment:
Is it enough for our further applications? I assume this would create `/` partition with 5 GB?

Reply:
Actually this, part is quite intended. However, I missed resizing it at the end, before uploading.
The idea was that we install the system on a small partition, and then resize the qcow2 image, leaving empty space at the end to factiliate both ZFS and QCOW2 deployments.